src/ansys/fluent/core/post_objects/post_object_definitions.py

```diff
@@ -43,7 +43,7 @@ def display(self, window_id: Optional[str] = None):
         Parameters
         ----------
         window_id : str, optional
-            Window id. If not specified unique id is used.
+            Window id. If not specified unique ID is used.
```

Comment:
```suggestion
            Window ID. If not specified a unique ID is used.
```

Reply:
@raph-luc  I missed `If not specified,` Done. Thank you.